local_pathfinding/ompl_path.py

```diff
@@ -97,6 +100,47 @@ def update_objectives(self):
         """
         raise NotImplementedError
 
+    def choose_planner(self, planner: str, si: ob.SpaceInformation) -> ob.Planner:
+        """Choose the planner to use for the OMPL query.
+
+        Args:
+            planner (str): Name of the planner to use.
+            si (ob.SpaceInformation): Encapsulates the planning problem to be solved.
+
+        Returns:
+            ob.Planner: Planner to use for the OMPL query.
+        """
+
+        match planner.lower():
+            case "bfmtstar":
+                return og.BFMT(si)
+            case "bitstar":
+                return og.BITstar(si)
+            case "fmtstar":
+                return og.FMT(si)
+            case "informedrrtstar":
+                return og.InformedRRTstar(si)
+            case "prmstar":
+                return og.PRMstar(si)
+            case "rrtstar":
+                return og.RRTstar(si)
+            case "sorrtstar":
+                return og.SORRTstar(si)
+            case "rrtxstatic":
+                return og.RRTXstatic(si)
+            case "rrtsharp":
+                return og.RRTsharp(si)
+            case "lazyprmstar":
+                return og.LazyPRMstar(si)
+            case "rrtconnect":
+                return og.RRTConnect(si)
+            case "lbtrrt":
+                return og.LBTRRT(si)
+            case "lazylbtrrt":
+                return og.LazyLBTRRT(si)
+            case _:
+                ou.OMPL_ERROR("Planner-type is not implemented in allocation function.")
```

Comment:
Use ros log error, printing what the given planner was and return one of the planners by default

Reply:
Made this global function so ros logger no longer accessible